adapter/src/components/styles/ErrorBoundary.style.js

```diff
@@ -29,6 +26,14 @@ export default css`
         justify-content: center;
     }
 
+    .fullscreen {
+        position: absolute;
+        top: 0px;
+        left: 0px;
+        right: 0px;
+        bottom: 0px;
+    }
+
```

Comment:
If the `fullscreen` prop is added to the `ErrorBoundary`, it will take up the whole screen instead